apiserver/facades/client/action/action.go

```diff
@@ -226,3 +251,13 @@ func (api *ActionAPI) WatchActionsProgress(ctx context.Context, actions params.E
 	}
 	return results, nil
 }
+
+// MakeActionResult does the actual type conversion from the operation domain
+// Action to params.ActionResult.
+func MakeActionResult(action operation.Action) params.ActionResult {
```

Comment:
todo: make private method. 

to: implement me. The required data from the call to Actions is known via the ActionResult struct.